File: Prefix-sum-SUBARRAY/1137-height-checker/height-checker.go
func heightChecker(heights []int) int {
	clonedHeights := make([]int, len(heights))
	copy(clonedHeights, heights)

	sort.Ints(clonedHeights)

	count := 0
	for i := 0; i < len(heights); i++ {
		if heights[i] != clonedHeights[i] {
			count++
		}
	}
	return count
}

File: Prefix-sum-SUBARRAY/1137-height-checker/height-checker.ts
function heightChecker(heights: number[]): number {
    const clonedHeights = [...heights];

    clonedHeights.sort((a, b) => a - b);

    let count = 0;
    for (let i = 0; i < heights.length; ++i) {
        if (heights[i] !== clonedHeights[i]) {
            count++;
        }
    }
    return count;
}
